scipy/special/tests/test_data.py

```diff
@@ -430,6 +430,8 @@ def test_boost():
         data(spherical_jn_, 'sph_bessel_data_ipp-sph_bessel_data', (0,1), 2, rtol=1e-13),
         data(spherical_yn_, 'sph_neumann_data_ipp-sph_neumann_data', (0,1), 2, rtol=8e-15),
 
+        data(owens_t, 'owenst_data_ipp-owenst', (0, 1), 2, rtol=5e-1),
```

Comment:
Min relative error 1.3031421275295663e-16 with absolute error 1.0842021724855044e-19
Max relative error 3.9470692224712853 with absolute error 2.319836168418231e-08
There is also examples with relative and absolute error = 0